unkown-packet.log

```diff
@@ -0,0 +1,17 @@
+Finishing quest with Sara:
```

Comment:
Prefer placing these debug/analysis comments in the relevant .go files to help keep things together in one place